src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordStatus.kt

```diff
@@ -137,6 +141,61 @@ enum class DiscordStatus(private val displayMessageSupplier: Supplier<String>?)
 
     CUSTOM({
         SkyHanniMod.feature.misc.discordRPC.customText.get() // custom field in the config
+    }),
+
+    AUTO({
+        val slayerResult = SLAYER.displayMessageSupplier!!.get()
+        val milestoneResult = try {
+            CROPMILESTONES.displayMessageSupplier!!.get()
+        }
+        catch (e: Exception){
+            "Unable to get milestone"
+        }
+        if (slayerResult != "Planning to do a slayer quest") slayerResult
```

Comment:
In the future we should not use the string check but rather a "slayer result type" enum